Package.swift

```diff
@@ -28,7 +28,7 @@ let package = Package(
         .package(url: "https://github.com/apple/FHIRModels.git", .upToNextMinor(from: "0.5.0")),
         .package(url: "https://github.com/StanfordSpezi/Spezi.git", .upToNextMajor(from: "1.8.1")),
         .package(url: "https://github.com/StanfordSpezi/SpeziFoundation.git", .upToNextMajor(from: "2.1.4")),
-        .package(url: "https://github.com/StanfordSpezi/SpeziHealthKit.git", .upToNextMajor(from: "1.0.1")),
+        .package(url: "https://github.com/StanfordSpezi/SpeziHealthKit.git", branch: "lukas/bulk-exporter"),
```

Comment:
TODO: update to SpeziHealthKit 1.0.2, once https://github.com/StanfordSpezi/SpeziHealthKit/pull/48 is merged and tagged